packages/mobile/src/screens/explore-screen/components/SearchExploreHeader.tsx

```diff
@@ -82,159 +111,122 @@ export const SearchExploreHeader = (props: SearchExploreHeaderProps) => {
     return () => keyboardDidHideListener?.remove()
   }, [navigation, params?.autoFocus])
 
-  // Focus the input when autoFocus is true and screen comes into focus
-  useFocusEffect(
-    useCallback(() => {
-      if (params?.autoFocus === true && textInputRef.current) {
-        const timer = setTimeout(() => {
-          textInputRef.current?.focus()
-        }, 100)
-
-        return () => clearTimeout(timer)
-      }
-    }, [params?.autoFocus])
-  )
-
-  // State
-
-  // Data fetching
-  const animatedFilterPaddingVertical = useSharedValue(spacing.l)
+  // Create derived values for better performance
+  const isAtTop = useDerivedValue(() => {
```

Comment:
Not familiar with this hook, what does it do? Prevent constantly reading layout values?